### PR TITLE
Implementation of EmailAddress form field type handler

### PR DESCRIPTION
**Pull request by MarioBlazek**
Implementation of EmailAddress form field type handler provided.

### CODE REVIEW

File: Form/Type/Email.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @return \eZ\Publish\Core\FieldType\EmailAddress\Value
+	 */
+	public function convertFieldValueFromForm( $data )
+	{
+		return new EmailAddress\Value( $data );
+	}
+}
```
Comment (pspanja) [inline Form/Type/Email.php]: No new line at the end of file.

File: Resources/config/form_fieldtype_handlers.yml [hunk 2]
```diff
@@ -24,3 +25,8 @@ services:
         class: %netgen.ezforms.form.fieldtype_handler.ezuser.class%
         tags:
             - {name: netgen.ezforms.form.fieldtype_handler, alias: ezuser}
+
+    netgen.ezforms.form.fieldtype_handler.ezemail:
+        class: %netgen.ezforms.form.fieldtype_handler.ezemail.class%
+        tags:
+            - {name: netgen.ezforms.form.fieldtype_handler, alias: ezemail}
```
Comment (pspanja) [inline Resources/config/form_fieldtype_handlers.yml]: No new line at the end of file.

File: Form/Type/Email.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+		$formBuilder->add( $fieldDefinition->identifier, "email", $options );
+	}
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @return string
+	 */
+	public function convertFieldValueToForm(Value $value)
```
Comment (pspanja) [inline Form/Type/Email.php]: CS: Missing space after opening and before closing parenthesis.

**Comment by pspanja** [comment]
You will need to convert tabs to spaces.
Aside from few other cosmetic issues, looks really good :)

**Comment by MarioBlazek** [comment]
Fixed all issues, tnx for review.

**Comment by pspanja** [comment]
Great, thanks for the first contribution here @MarioBlazek :)

**Pull request merged**